### ISSUE TITLE
Getting error 001003 (42000): SQL compilation error : syntax error line 6 at position 17 unexpected '<EOF>'.

### ISSUE THREAD

**Issue by hkkachhia** [sqitchers/sqitch#737]
Hello, 

I am getting error Getting error 001003 (42000): SQL compilation error : syntax error line 6 at position 17 unexpected '<EOF>'.
when deploying store procedure.  

The store procedure is working fine when I am executed on worksheet of snowflake portal . I am unable to find what is the issue with my store procedure.

**Comment by theory** [comment]
Sounds like the SnowSQL client is hitting something it can't parse. Can you post an example here?

**Comment by hkkachhia** [comment]
Thanks for the replied. 

I have found the issue and fix it.  As I have mentioned in my question that there is no error in Store procedure SQL script so that is correct. But the issue is that sqitch using classing console to run the script. So there is some syntax differ to run store procedure in classic console. 

Here is the example of : 

If you are using [SnowSQL](https://docs.snowflake.com/en/user-guide/snowsql.html) or the [Classic Console](https://docs.snowflake.com/en/user-guide/ui-using.html), use this example instead (see [Using Snowflake Scripting in SnowSQL and the Classic Console](https://docs.snowflake.com/en/developer-guide/snowflake-scripting/running-examples.html)):

CREATE OR REPLACE PROCEDURE output_message(message VARCHAR)
RETURNS VARCHAR NOT NULL
LANGUAGE SQL
AS
$$
BEGIN
  RETURN message;
END;
$$
;

**Comment by theory** [comment]
Yes, Sqitch uses [SnowSQL](https://docs.snowflake.com/en/user-guide/snowsql.html) to run scripts, so you will need to write things formatted for that utility.
